[2022-06-04 18:55:16,842] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 manual__2022-06-04T18:54:56.996236+00:00 [queued]>
[2022-06-04 18:55:16,921] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 manual__2022-06-04T18:54:56.996236+00:00 [queued]>
[2022-06-04 18:55:16,930] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 18:55:16,931] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-04 18:55:16,931] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-04 18:55:17,033] {taskinstance.py:1378} INFO - Executing <Task(BashOperator): task_4> on 2022-06-04 18:54:56.996236+00:00
[2022-06-04 18:55:17,053] {standard_task_runner.py:52} INFO - Started process 37383 to run task
[2022-06-04 18:55:17,084] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', 'manual__2022-06-04T18:54:56.996236+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpsvwbivaq', '--error-file', '/tmp/tmpzhpuj_ef']
[2022-06-04 18:55:17,089] {standard_task_runner.py:80} INFO - Job 10: Subtask task_4
[2022-06-04 18:55:17,644] {task_command.py:370} INFO - Running <TaskInstance: parallel_dag.task_4 manual__2022-06-04T18:54:56.996236+00:00 [running]> on host airflowvm
[2022-06-04 18:55:18,152] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2022-06-04T18:54:56.996236+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-04T18:54:56.996236+00:00
[2022-06-04 18:55:18,160] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-04 18:55:18,164] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-06-04 18:55:18,218] {subprocess.py:85} INFO - Output:
[2022-06-04 18:55:21,238] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-04 18:55:21,369] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20220604T185456, start_date=20220604T185516, end_date=20220604T185521
[2022-06-04 18:55:21,500] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-04 18:55:21,688] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
